environment.conf.json

```diff
@@ -1,6 +1,10 @@
 {
+  "siteTitle": "SIA - Signalen Informatievoorziening Amsterdam",
   "logoUrl": null,
+  "logoWidth": 0,
   "logoHeight": 0,
+  "logoWidthSmall": 0,
+  "logoHeightSmall": 0,
```

Comment:
Wouldn't it be better to have these props be nested in a `logo` prop in this file?

Reply:
I have been thinking about that too. Will do!